gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -764,12 +865,13 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
         }
 
         prop->input_meta.num_tensors = num_dims;
+      } else if (value) {
+        /** Once configured, it cannot be changed in runtime for now */
+        g_assert (!prop->input_configured);
```

Comment:
@myungjoo is this assert needed or this can be just a warning?

Reply:
We should prohibit set-prop on input dimension if it's already configured (GST-CAP negotiation completed).

It's fine to ignore configuring the value with "VISIBLE" warning; the warning level should be HIGH. (use @jaeyun-jung's recent log APIs)

In fact, we need to limit g_assert for internal error only for higher code quality. External errors (errors incurred by users and applications) should not do assert, but emit proper error messages (visiable and explainable).

Reply:
Replaced `g_assert` with meaningful comments.

> HIGHEST for the first few violations and WARN for the rest or for a few seconds for the same errors; for throttling, we will need to upgrade the logging mechanism in the future.

That is a good suggestion and should be done.